From 75d3c74f05c1260df71e5852dab5d7f0c8c4bf85 Mon Sep 17 00:00:00 2001
From: Ronald Wampler <rdwampler@gmail.com>
Date: Sat, 19 Sep 2015 10:14:03 -0400
Subject: [PATCH] =?UTF-8?q?OFBOutlineView:=20Rename=20draggedItems=20?=
 =?UTF-8?q?=E2=86=92=20draggedOFBItems?=
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit
Upstream-Status: Backport

This property's name was conflicting with a private instance variable in our superclass (NSOutlineView) for the 10.11 SDK.
---
 Frameworks/OakFileBrowser/src/ui/OFBOutlineView.mm | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/Frameworks/OakFileBrowser/src/ui/OFBOutlineView.mm b/Frameworks/OakFileBrowser/src/ui/OFBOutlineView.mm
index caaeb5d..487c5c0 100644
--- ./Frameworks/OakFileBrowser/src/ui/OFBOutlineView.mm
+++ ./Frameworks/OakFileBrowser/src/ui/OFBOutlineView.mm
@@ -19,7 +19,7 @@ @interface OFBOutlineView ()
 	NSSize defaultIntercellSpacing;
 	NSColor* defaultBackgroundColor;
 }
-@property (nonatomic) NSArray* draggedItems;
+@property (nonatomic) NSArray* draggedOFBItems;
 
 - (void)performDoubleClick:(id)sender;
 @end
@@ -210,15 +210,15 @@ - (NSImage*)dragImageForRowsWithIndexes:(NSIndexSet*)anIndexSet tableColumns:(NS
 		if(id item = [self itemAtRow:index])
 			[items addObject:item];
 	}
-	self.draggedItems = items;
+	self.draggedOFBItems = items;
 	return [super dragImageForRowsWithIndexes:anIndexSet tableColumns:anArray event:anEvent offset:aPointPointer];
 }
 
 - (void)draggedImage:(NSImage*)anImage endedAt:(NSPoint)aPoint operation:(NSDragOperation)aDragOperation
 {
-	if(self.draggedItems && [self.dataSource respondsToSelector:@selector(outlineView:draggedItems:endedWithOperation:)])
-		[(id <FSDataSourceDragSource>)self.dataSource outlineView:self draggedItems:self.draggedItems endedWithOperation:aDragOperation];
-	self.draggedItems = nil;
+	if(self.draggedOFBItems && [self.dataSource respondsToSelector:@selector(outlineView:draggedItems:endedWithOperation:)])
+		[(id <FSDataSourceDragSource>)self.dataSource outlineView:self draggedItems:self.draggedOFBItems endedWithOperation:aDragOperation];
+	self.draggedOFBItems = nil;
 
 	if([NSOutlineView respondsToSelector:@selector(draggedImage:endedAt:operation:)])
 		[super draggedImage:anImage endedAt:aPoint operation:aDragOperation];
